test/functional/settings-e2e-specs.js

```diff
@@ -0,0 +1,43 @@
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import _ from 'lodash';
+import AndroidDriver from '../..';
+import DEFAULT_CAPS from './desired';
+import { sleep } from 'asyncbox';
+
+chai.should();
+chai.use(chaiAsPromised);
+
+let defaultCaps = _.defaults({
+  androidInstallTimeout: 90000,
+  browserName: 'chrome'
+}, DEFAULT_CAPS);
+
+describe.skip('toggle wifi tests', () => {
```

Comment:
We really ought to set `process.env.REAL_DEVICE` as false when `process.env.TRAVIS` is true.